creator-node/scripts/start.sh

```diff
@@ -64,9 +67,18 @@ if [ -z "$dbUrl" ]; then
 fi
 
 if [[ "$devMode" == "true" ]]; then
-    ./node_modules/.bin/nodemon --inspect=0.0.0.0:${debuggerPort} src/index.js | tee >(logger) | ./node_modules/.bin/bunyan
+    if [ "$link_libs" = true ]
+    then
+        cd ../audius-libs
+        npm link
+        cd ../app
+        npm link @audius/libs
+        ./node_modules/.bin/nodemon --watch src/ --watch ../audius-libs/ src/index.ts | tee >(logger) | ./node_modules/.bin/bunyan
+    else
+        ./node_modules/.bin/nodemon --watch src/ src/index.ts | tee >(logger) | ./node_modules/.bin/bunyan
+    fi
 else
-    node src/index.js | tee >(logger)
+    node build/src/index.js | tee >(logger)
```

Comment:
what is `(logger)` evaluated to here?

Reply:
thats a good question, i think @cheran-senthil is best suited to answer this one

Reply:
`logger` is a system level command (part of syslog)

Here we use tee to send the output of these commands as input to the `logger` command and stdout